up42/utils.py

```diff
@@ -366,16 +366,19 @@ def autocomplete_order_parameters(order_parameters: dict, schema: dict):
     order_parameters["params"] = dict(order_parameters["params"], **additional_params)
 
     # Log message help for parameter selection
-    for param in additional_params.keys():
-        if param in ["aoi", "geometry"]:
-            continue
-        elif "allOf" in schema["properties"][param]:  # has further definitions key
-            potential_values = [x["id"] for x in schema["definitions"][param]["enum"]]
-            logger.info(f"As `{param}` select one of {potential_values}")
-        else:
-            # Full information for simple parameters
-            del schema["properties"][param]["title"]
-            logger.info(f"As `{param}` select `{schema['properties'][param]}`")
+    try:
+        for param in additional_params.keys():
+            if param in ["aoi", "geometry"]:
+                continue
+            elif "allOf" in schema["properties"][param]:  # has further definitions key
+                potential_values = [x["const"] for x in schema["definitions"][param]["anyOf"]]
```

Comment:
From JSON schema's point of view, the ["anyOf"](https://json-schema.org/understanding-json-schema/reference/combining.html#anyof) could mave many possible configs. Is tasking guarantees that it won't be anything else?

Reply:
There is no such guarantee.
This entire function will be updated. See [this ticket](https://up42.atlassian.net/browse/INT-198).